### DISCUSSION TITLE
Run msbuild on top of GPU

### DISCUSSION THREAD

**Discussion by venujakasun** [dotnet/msbuild#7728] [Q&A]
Is there any possibility to use GPU for build instead of CPU?

**Comment by rainersigwald** [comment]
I'm not aware of any build engine that is GPU aware, and I don't think the kinds of compute MSBuild does is a great fit for the massively parallel execution environment. Some compilers are experimenting with GPU work but that would happen outside is MSBuild's control.


We're always interested in new developments but I wouldn't expect much on this front in particular.
